outdated/modules/httpd.reb

```diff
@@ -82,24 +97,24 @@ sys/make-scheme [
                         close client
                     ]
 
-                    true [
-                        client
-                    ]
+                    default [client]
                 ]
             ]
 
-            close [
+            'close [
                 close client
             ]
 
-            (
-                net-utils/net-log ["Unexpected Client Event: " uppercase form event/type]
+            default [
+                net-utils/net-log [
+                    "Unexpected Client Event:" uppercase form event/type
+                ]
                 client
-            )
+            ]
         ]
     ]
 
-    init: func [server [port!] /local spec][
```

Comment:
Stylistic change: definitely don't use `/local` as these are refinements which are visible in the interface... use `<local>` instead.  Also for the module repository, try to minimize usage of FUNC.